repos/system_upgrade/common/actors/trackcustommodifications/libraries/trackcustommodifications.py

```diff
@@ -0,0 +1,92 @@
+import ast
+import os
+
+from leapp.exceptions import StopActorExecution
+from leapp.libraries.common.config.version import get_source_major_version, get_target_major_version
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import CustomModifications
+
+RPMS_TO_CHECK = ['leapp-upgrade-el{source}toel{target}']
+DIRS_TO_SCAN = ['/usr/share/leapp-repository']
+
+
+def _get_rpms_to_check():
+    return [rpm.format(source=get_source_major_version(), target=get_target_major_version()) for rpm in RPMS_TO_CHECK]
```

Comment:
I will look into this, thanks!

Reply:
I like the notion of DRY, but the original issue was strictly about modified actor code only stressing out changed file -> actor name mapping. This will introduce the need to filter-out snactor packages (I don't see any use of tracking any changes there as it will be just poisoning the output) and even if we have el9toel10 in the future my implementation won't need to be updated.
So not really sure this is the way.

Reply:
Yep, I see that tracking leapp code changes is also part of this. Then `get_leapp_packages` makes sense, will refactor, thanks.

Reply:
resolved.